From 148568b5c36821e2f5e26771e88b7c15bc4ae216 Mon Sep 17 00:00:00 2001
From: Alessio Igor Bogani <alessio.bogani@elettra.eu>
Date: Wed, 7 Aug 2019 18:45:43 +0200
Subject: [PATCH 3/3] powerpc/embedded6xx: Make reboot works on MVME5100

The mtmsr() function hangs during restart. Make reboot works on
MVME5100 removing that function call.

Signed-off-by: Alessio Igor Bogani <alessio.bogani@elettra.eu>
---
 arch/powerpc/platforms/embedded6xx/mvme5100.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/powerpc/platforms/embedded6xx/mvme5100.c b/arch/powerpc/platforms/embedded6xx/mvme5100.c
index 83f72723e2f1..95676a85c030 100644
--- a/arch/powerpc/platforms/embedded6xx/mvme5100.c
+++ b/arch/powerpc/platforms/embedded6xx/mvme5100.c
@@ -183,9 +183,7 @@ static void mvme5100_show_cpuinfo(struct seq_file *m)
 
 static void __noreturn mvme5100_restart(char *cmd)
 {
-
 	local_irq_disable();
-	mtmsr(mfmsr() | MSR_IP);
 
 	out_8((u_char *) restart, 0x01);
 
-- 
2.17.1

